### ISSUE TITLE
Error on available_players() function: 'unsupported operand type(s) for /: 'NoneType' and 'float''

### ISSUE THREAD

**Issue by mhbw** [jaebradley/draftkings_client#61]
found this trying to use the available_players function, it seems that the same player group is being read incorrectly as compared to other segments. 

for example, I ran 
```

from draft_kings.client import draft_group_details

draft_group_details(draft_group_id=41643)
```

and that worked fine, but 
```

from draft_kings.client import available_players

available_players(draft_group_id=41643)
```

gets 
`
TypeError: unsupported operand type(s) for /: 'NoneType' and 'float'`

**Comment by jaebradley** [comment]
@mhbw did you see this issue: https://github.com/jaebradley/draftkings_client/issues/37

This should be fixed in [v2.0.2](https://github.com/jaebradley/draftkings_client/releases/tag/2.0.2)

**Comment by mhbw** [comment]
hello @jaebradley ; thanks for the quick response and the great package; I did see that in the closed issues, but was still seeing the problem. 

this is maybe an interesting side problem; I tried updating with pip and it said I had the most recent package, v2.0.1, which obviously is not the most recent. I tried to force the update with

````
 pip install -Iv draft_kings==2.0.2
````

and got back

````
ERROR: Could not find a version that satisfies the requirement draft_kings==2.0.2 (from versions: 2.0.2.macosx-10.15-x86_64, 0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 2.0.0, 2.0.1)
ERROR: No matching distribution found for draft_kings==2.0.2
````

**Comment by jaebradley** [comment]
@mhbw hmmm are you running the `pip install` on a mac machine or a non-mac machine?

I think I might know what the problem is, but looking for clarification based on your response.

**Comment by mhbw** [comment]
yep, on a Mac

**Comment by jaebradley** [comment]
@mhbw sorry about the super-delayed response - I've been working on pushing out v3 of this client and haven't had time to respond to this issue.

I merged #42 (which fixed some vulnerability issues associated with the `urllib3` library that this client depends on) and published `v2.0.3`.

I _think_ this should `pip install` correctly.

Here is a working `repl.it` for `v2.0.3`.

https://repl.it/@jaebradley/Countries-1#main.py

![image](https://user-images.githubusercontent.com/8136030/101290433-4d079580-37d0-11eb-9762-42d9a7dfd158.png)
![image](https://user-images.githubusercontent.com/8136030/101290442-63155600-37d0-11eb-8850-67382cd0e219.jpeg)
